frontend/src/common/types/button/button.ts

```diff
@@ -2,4 +2,5 @@ export type ButtonPropsType = {
   className?: string;
   text: string;
   onClick?: (e: React.MouseEvent<HTMLButtonElement>) => void;
+  type?: 'button' | 'submit' | 'reset' | undefined;
```

Comment:
Why would we need a `reset` type, it's not part of the HTML button types? And the `undefined` shouldn't be explicitly specified if you use the `?`. It's a duplication